src/daft-sql/src/modules/aggs.rs

```diff
@@ -108,6 +108,7 @@ pub fn to_expr(expr: &AggExpr, args: &[ExprRef]) -> SQLPlannerResult<ExprRef> {
             Ok(args[0].clone().sum())
         }
         AggExpr::ApproxCountDistinct(_) => unsupported_sql_err!("approx_percentile"),
+        AggExpr::ApproxDistinct(_) => unsupported_sql_err!("approx_percentile"),
```

Comment:
There seems to be a mismatch between expression name and error string

Reply:
Resolved for the line above as well.